### ISSUE TITLE
Fix the Username Field, to let users make new custom Names

### ISSUE THREAD

**Issue by IamTregsthedev** [project-lumiere/lumiere.codes#82]
The Username field, does not let people to change their username propley.

**See Video **

It lets them to add new things and backspace from their current username

You can fix this by doing Ctrl + A (command + A on Mac) and typing a new one. (However, that's not very user friendly)

https://www.loom.com/share/eb07df72089741cdb1aad89947ebdf8d

**Comment by arashnrim** [comment]
Good catch! There's something definitely not right with how the input works.

If no one else wants to take this up, may I? I can have a go at this after working on mobile responsiveness. 😜

**Comment by IamTregsthedev** [comment]
Any change on this?
